server/app/controllers/admin/AdminApplicationController.java

```diff
@@ -294,6 +293,7 @@ public Result show(Http.Request request, long programId, long applicationId)
             .join();
     ImmutableList<Block> blocks = roApplicantService.getAllActiveBlocks();
     ImmutableList<AnswerData> answers = roApplicantService.getSummaryData();
+    var statusDefinitions = program.statusDefinitions();
```

Comment:
Nit: This variable is used from a single place. Consider just inlining the call for readability. As it stands, I don't think there's readability benefit since the variable name is the same as the method being called. I'd make the same argument for programId and programName. That said, I can also see an argument for doing it here since everything else is initialized before the call. Up to you.

Reply:
inlined this one, Kept the others as I should have used programName in another spot.  programId is a method param, so tricky to "use the right one" if we want to pull from the definition.